packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -101,6 +108,25 @@ export class DeltaScheduler {
                     this.schedulingLog.numberOfTurns++;
                     this.schedulingLog.totalProcessingTime += elapsedTime;
```

Comment:
I'd recommend naming these two variables differently, i.e. find better name (fort first one) - it's confusing to have exactly same names with quite different meaning / semantics:
this.totalProcessingTime
this.schedulingLog.totalProcessingTime

Reply:
Great point - changed the this.totalProcessingTime to currentAllowedProcessingTimeForTurn